pkg/controllers/support.go

```diff
@@ -155,3 +155,10 @@ func (r *RayClusterReconciler) isRayDashboardOAuthEnabled() bool {
 	}
 	return true
 }
+
+func (r *RayClusterDefaulter) isRayDashboardOAuthEnabledWebhook() bool {
```

Comment:
Would it be possible to use the existing `isRayDashboardOAuthEnabled` [function](https://github.com/project-codeflare/codeflare-operator/blob/8b3b27c9ea4989a089de953aa2030cb165393d53/pkg/controllers/support.go#L152) here? and the RayClusterReconciler in general rather than the RayClusterDefaulter?